build/yaml/crd/nsx.vmware.com_subnets.yaml

```diff
@@ -56,11 +56,13 @@ spec:
                     type: boolean
                 type: object
               accessMode:
+                default: Private
                 description: Access mode of Subnet, accessible only from within VPC
                   or from outside VPC.
                 enum:
                 - Private
                 - Public
+                - Project
```

Comment:
wcp changed this to PRIVATE_TGW, @lxiaopei shall this type also needs to change

Reply:
from vpc 2.0 doc, they use "Private_TGW“.